django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
From a security stand point, argon2 is better than PBKDF2 because it's memory hard as well as CPU hard. Security wise argon2 > scrypt > PBKDF2 ~= bcrypt.